plugins/kubernetes/app/models/kubernetes/role_config_file.rb

```diff
@@ -0,0 +1,173 @@
+module Kubernetes
+
+  # This file represents a Kubernetes configuration file for a specific project role.
+  # A single configuration file can have both a Replication Controller and a Service, as two separate documents.
+  #
+  # For the schema definition of a Replication Controller see:
+  # https://cloud.google.com/container-engine/docs/replicationcontrollers/operations
+  #
+  # For the schema definition of a Service, see:
+  # https://cloud.google.com/container-engine/docs/services/operations
+  #
+  class RoleConfigFile
```

Comment:
I know. We'll probably need to review ReleaseDoc when we get there (i.e., when we start dealing with the actual release logic). For now, the main goal for this class is just to represent a specific Role configuration file. At the moment, it only has logic to parse and retrieve data, but, in the mid-term, It could also contain some validation logic as well, but we haven't thought about that yet. Where and how to validate the yml files.

We could then leverage this class in the ReleaseDoc, when creating a Release. The logic to parse the file is already encapsulated in here, so, we just need to refactor the logic that deals with overriding the template with the specifics of the release.